tests/spread/plugins/go/classic/task.yaml

```diff
@@ -1,32 +1,24 @@
 summary: Build and run a basic Go snap
 
-# Classic snaps don't build on 14.04
-systems: [-ubuntu-14*]
-
 environment:
   SNAP_DIR: ../snaps/go-gotty
 
 prepare: |
+  . "$TOOLS_DIR/snapcraft-yaml.sh"
+  set-base "$SNAP_DIR/snap/snapcraft.yaml"
   apt install patchelf -y
```

Comment:
This isn't available in Trusty. You can either leave the `systems` filter there but change the comment, or use `apt install patchelf -y || true` and only check the rpaths `if which patchelf &> /dev/null`.

Reply:
the filter is at the top level in spread.yaml though and only special casing legacy.

Reply:
Trusty is dead when dealing with bases btw; it should only ever go into that mode with `--destructive` and at that point, all bets are off.